modules/qtwidgets/src/inviwoqtutils.cpp

```diff
@@ -589,6 +589,19 @@ QString windowTitleHelper(const QString& title, const QWidget* widget) {
     return cap;
 }
 
+int refSpacePx(const QWidget* w) { return emToPx(w, refSpaceEm()); }
+
+QSize emToPx(const QWidget* w, QSizeF ems) {
+    return QSize{emToPx(w, ems.width()), emToPx(w, ems.height())};
+}
+
+int emToPx(const QWidget* w, double em) { return emToPx(w->fontMetrics(), em); }
```

Comment:
need to ensure that the widget is polished before accessing the font metrics.
`w->ensurePolished();` 

see https://doc.qt.io/qt-5/qwidget.html#ensurePolished

Reply:
Was considering that at so e point but forgot to add it since it sems to work anyway...